flight/Engine.php

```diff
@@ -618,26 +613,22 @@ public function _halt(int $code = 200, string $message = ''): void
         }
     }
 
-    /**
-     * Sends an HTTP 404 response when a URL is not found.
-     */
+    /** Sends an HTTP 404 response when a URL is not found. */
     public function _notFound(): void
     {
+        $output = '<h1>404 Not Found</h1><h3>The page you have requested could not be found.</h3>';
```

Comment:
if you wanted to be mega fancy you could do
```php
$output = <<<HTML
    <h1>404 Not Found</h1><h3>The page you have requested could not be found.</h3>
    HTML;
```
and then you'd have syntax highlighting ;)

Reply:
I extracted a variable just to reduce the line size and phpcs will stop complaining. But I also like heredoc for these things